src/sentry/static/sentry/less/group-detail.less

```diff
@@ -2802,3 +2802,9 @@ ul.crumbs {
     display: none;
   }
 }
+
+.issue-modal {
+  .modal-content {
+    overflow: visible;
+  }
+}
```

Comment:
😬 will let @Chrissy or @billyvg decide if we want to just remove it globally or not

Reply:
I think we should remove globally - I'd rather avoid this problem for future features than worry about any potential regressions.

Reply:
ok, updated!

Reply:
agree with billy.

there is a decent chance this was used as a clearfix, in which case you could do `overflow: auto` but that might inject random scrollbars into places so probably just remove it.